components/Heading/Heading.css

```diff
@@ -5,91 +5,79 @@
 }
```

Comment:
Paragraph styles weren't updated but also contain this issue

Reply:
And this 👍🏻